setup.py

```diff
@@ -155,29 +67,9 @@ def get_cuda_version() -> Version:
         txt = subprocess.check_output([nvcc, "--version"], text=True)
         return Version(re.findall(r"release (\d+\.\d+),", txt)[0])
 
-    class NinjaBuildExtension(torch_cpp_ext.BuildExtension):
-        def __init__(self, *args, **kwargs) -> None:
-            # do not override env MAX_JOBS if already exists
-            if not os.environ.get("MAX_JOBS"):
-                max_num_jobs_cores = max(1, os.cpu_count())
-                os.environ["MAX_JOBS"] = str(max_num_jobs_cores)
```

Comment:
Do we have similar logic to restrict `MAX_JOBS` now?

Reply:
In `cpp_ext.py`, I call `_get_num_workers()` from PyTorch cpp extension, which checks `MAX_JOBS` env var.